src-electron/rest/initialize.js

```diff
@@ -35,9 +36,15 @@ const fsp = fs.promises
  */
 function sessionAttempt(db) {
   return async (req, res) => {
-    if (req.body.search?.includes('filePath=%2F')) {
-      let filePath = req.body.search.split('filePath=')
-      filePath = filePath[1].replaceAll('%2F', '//').trim()
+    let search = req.body.search
+    let filePath
+    if (search[0] === '?') {
+      search = search.substring(1)
+    }
+    let query = querystring.parse(search)
+    if (query.filePath) {
+      filePath = query.filePath
```

Comment:
Does querystring.parse handle the "replaceAll('%2F', '//')" automatically here?

Reply:
yes